lib/resources/example_json/r4_examples_insurance_plan.rb

```diff
@@ -0,0 +1,180 @@
+# frozen_string_literal: true
+
+module Cerner
+  module Resources
+    R4_INSURANCEPLAN_BUNDLE ||= {
+      'resourceType': 'Bundle',
+      'id': 'e445217e-122b-4e89-9ae3-74e0ba1c24fb',
+      'type': 'searchset',
+      'total': 4,
+      'link': [
+        {
+          'relation': 'self',
+          'url': 'https://fhir-ehr-latest.sandboxcerner.com/r4/dacc6494-e336-45ad-8729-b789ff8663c6/InsurancePlan?owned-by=589783'
```

Comment:
Instead of sandbox can you update the url to `https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d` at all places

Reply:
Changed throughout the changes https://github.com/cerner/fhir.cerner.com/pull/809/commits/b36ff1fe0e82d264abd1006b6c29a6cace1354b9#diff-505b119a76d7cbd4b9ad19f605bcfd1d674bfb0fca50632ebc7948b1bc50b3caR52

Reply:
@NimishaMN, are we no longer using the open sandbox url on fhir.cerner.com? https://fhir-ehr-code.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d

Reply:
It is suggested that we provide the Prod urls with her reference since the fhir.cerner.com is a client facing Doc and they mainly use C1941.

Reply:
I was just looking at patient and encounter resources and they use https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d for their GET calls. Did something change?

Reply:
@alexbeckner1  i believe Manideep has updated the URLs from "-latest.sandbox" to C1941's URLs and that's what is needed. Hence this comment it resolved. You both are correct. Our open sandbox is C1941 domain(which is having prod replica code version)

Reply:
i believe it's sorted out now and we encourage using "https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d" as the endpoints

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/809/commits/b15a0dd161ba774ee4a02ba6c8fc2d398615a7f3#diff-505b119a76d7cbd4b9ad19f605bcfd1d674bfb0fca50632ebc7948b1bc50b3caR52

Reply:
+1